tests/micro/arduino/test_arduino_rpc_server.py

```diff
@@ -0,0 +1,352 @@
+import datetime
+import os
+import pathlib
+import shutil
+import sys
+import time
+
+import numpy as np
+import onnx
+from PIL import Image
+import pytest
+import tflite
+
+import tvm
+from tvm import micro, relay
+from tvm.relay.testing import byoc
+
+import conftest
+
+"""
+This unit test simulates an autotuning workflow, where we:
+1. Instantiate the Arduino RPC server project
+2. Build and flash that project onto our target board
+
+"""
+
+# We'll make a new workspace for each test
+@pytest.fixture(scope="function")
+def workspace_dir(platform, run_hardware_tests):
+    if not run_hardware_tests:
```

Comment:
can you make this a decorator like tvm.testing.requires_micro? just declare the decorator in conftest.py